doc/nrf/releases_and_maturity/releases/release-notes-2.9.0.rst

```diff
@@ -0,0 +1,882 @@
+
+    * :ref:`direct_test_mode`
+    * :ref:`peripheral_hids_mouse`
+    * :ref:`peripheral_lbs`
+    * :ref:`power_profiling`
+    * :ref:`peripheral_uart`
+
+* Updated:
+
+  * Configurations of the following Bluetooth samples to make the :ref:`Zephyr Memory Storage (ZMS) <zephyr:zms_api>` the default settings backend for all board targets that use the MRAM technology:
+
+      * :ref:`bluetooth_central_hids`
+      * :ref:`peripheral_hids_keyboard`
+      * :ref:`peripheral_hids_mouse`
+      * :ref:`central_and_peripheral_hrs`
+      * :ref:`central_bas`
+      * :ref:`central_nfc_pairing`
+      * :ref:`central_uart`
+      * :ref:`peripheral_bms`
+      * :ref:`peripheral_cgms`
+      * :ref:`peripheral_cts_client`
+      * :ref:`peripheral_lbs`
+      * :ref:`peripheral_mds`
+      * :ref:`peripheral_nfc_pairing`
+      * :ref:`power_profiling`
+      * :ref:`peripheral_rscs`
+      * :ref:`peripheral_status`
+      * :ref:`peripheral_uart`
+      * :ref:`ble_rpc_host`
+
+    As a result, all :ref:`zephyr:nrf54h20dk_nrf54h20` configurations of the affected samples were migrated from the NVS settings backend to the ZMS settings backend.
+  * Testing steps in the :ref:`peripheral_hids_mouse` to provide the build configuration that is compatible with the `Bluetooth Low Energy app`_ testing tool.
+
+  * :ref:`power_profiling` sample that now has added support for the :ref:`zephyr:nrf54h20dk_nrf54h20` board target.
+
+Bluetooth Fast Pair samples
+---------------------------
+
+* :ref:`fast_pair_locator_tag` sample:
+
+  * Updated the MCUboot bootloader configuration for the :ref:`zephyr:nrf54l15dk_nrf54l15` board target to enable the :kconfig:option:`CONFIG_FPROTECT` Kconfig option that is used to protect the bootloader partition against memory corruption.
+
+* :ref:`fast_pair_input_device` sample:
+
+  * Added support for the :ref:`zephyr:nrf54h20dk_nrf54h20` board target.
+
+Bluetooth Mesh samples
+----------------------
+
+* Added:
+
+  * Support for nRF54L15 in the following samples:
+
+    * :ref:`bluetooth_mesh_sensor_client`
+    * :ref:`bluetooth_mesh_sensor_server`
+    * :ref:`bluetooth_ble_peripheral_lbs_coex`
+    * :ref:`bt_mesh_chat`
+    * :ref:`bluetooth_mesh_light_switch`
+    * :ref:`bluetooth_mesh_silvair_enocean`
+    * :ref:`bluetooth_mesh_light_dim`
+    * :ref:`bluetooth_mesh_light`
+    * :ref:`ble_mesh_dfu_target`
+    * :ref:`bluetooth_mesh_light_lc`
+    * :ref:`ble_mesh_dfu_distributor`
+
+  * Support for nRF54L05 in the following samples:
+
+    * :ref:`bluetooth_mesh_sensor_client`
+    * :ref:`bluetooth_mesh_sensor_server`
+    * :ref:`bluetooth_ble_peripheral_lbs_coex`
+    * :ref:`bt_mesh_chat`
+    * :ref:`bluetooth_mesh_light_switch`
+    * :ref:`bluetooth_mesh_silvair_enocean`
+    * :ref:`bluetooth_mesh_light_dim`
+    * :ref:`bluetooth_mesh_light`
+    * :ref:`bluetooth_mesh_light_lc`
```

Comment:
Sorry bu we need to revert those changes for 2.9:
https://github.com/nrfconnect/sdk-nrf/pull/19378#pullrequestreview-2492360120

So no entry for Bluetooth mesh samples.